.eslintrc

```diff
@@ -17,7 +17,8 @@
         "semi": [
             2,
             "always"
-        ]
+        ],
+        "no-use-before-define": 0
```

Comment:
I like to write functions in the order they are used rather than the order they are defined. Just a personal preference. Happy to reorder them if we should remove this.